packaging/rpm/microshift.spec

```diff
@@ -165,6 +165,26 @@ The microshift-multus-release-info package provides release information files fo
 release. These files contain the list of container image references used by
 the Multus CNI for MicroShift and can be used to embed those images into osbuilder blueprints.
 
+%package flannel
```

Comment:
Can we make this NOT built by default, unless a special `make` command is specified?

We should be able to disable the default build using `%bcond_with / %bcond_without` or similar.

Reply:
@ggiguash Is it because we don't want to make it downstream?

Reply:
Yes, exactly.

Reply:
@ggiguash Added new comment which will take `WITH_FLANNEL` as env to make decision.